back/spec/acceptance/projects_allowed_input_topics_spec.rb

```diff
@@ -24,6 +24,15 @@
       json_response = json_parse(response_body)
       expect(json_response[:data].size).to eq 1
     end
+
```

Comment:
@adessy closing this PR but do you get why the test above this suddenly started failing after adding the one below? 🤔